src/modules/vtol_att_control/standard.cpp

```diff
@@ -137,20 +137,13 @@ void Standard::update_vtol_state()
 			_mc_yaw_weight = 1.0f;
 			_mc_throttle_weight = 1.0f;
 
-		} else if (_vtol_schedule.flight_mode == FW_MODE) {
+		} else if (_vtol_schedule.flight_mode == FW_MODE ||
+			   _vtol_schedule.flight_mode == TRANSITION_TO_FW) {
 			// transition to mc mode
 			_vtol_schedule.flight_mode = TRANSITION_TO_MC;
 			_flag_enable_mc_motors = true;
 			_vtol_schedule.transition_start = hrt_absolute_time();
 
-		} else if (_vtol_schedule.flight_mode == TRANSITION_TO_FW) {
```

Comment:
We need this, it's important should the transition go really wrong (excessive tilt/roll for some reason). Doing the normal back transition might be fatal in this case. I guess we need a soft abort for the transition timeout. I'm working on a slowdown during the back transition which could always be active in auto modes, and we remove the RTL on transition failsafe for manual modes (it doesn't make sense there anyway).